### PR TITLE
Add Name text & Healthbar offset entries in characters.json

### PR DESCRIPTION
**Pull request by AtlantaDancer**
I had the thought after we wrapped up last night to put the offsets needed for the name display in the json file.  Here's the first pass at entries for them.  The settings for each character will need be tested & determined, but this gets us started.

### CODE REVIEW

File: game/src/assets/json/characters.json [hunk 1]
```diff
@@ -7,18 +7,18 @@
     "projectile": {
       "type": "light",
       "baseRange": 1000,
-      "fireOffset": {"x": 0, "y": -150}
+      "fireoffset": {"x": 0, "y": -150}
```
Comment (ignacioxd) [inline game/src/assets/json/characters.json]: Oops, looks like a find-and-replace mishap :)

File: game/src/assets/json/characters.json [hunk 2]
```diff
@@ -12,7 +12,13 @@
     "collider": {
       "size": 70,
       "offset": {"x": 87, "y": 80}
-    }
+    },
+    "NameText": {
+	  "Offset": {"x": 0, "y": -40}
```
[the remark this answers is not in the document]
Comment (ignacioxd) [reply game/src/assets/json/characters.json]: Don't forget this last bit about percentages. It's the most important part about this review.

**Comment by AtlantaDancer** [comment]
This should be good now.  Character uses json values and modified offsets for each character.

File: game/src/assets/json/characters.json (hunk 1 above)
Comment (ignacioxd) [reply game/src/assets/json/characters.json]: This still needs to be addressed.

**Pull request merged**